packages/util/src/logger/cloud-sdk-logger.ts

```diff
@@ -203,6 +203,29 @@ export function getGlobalLogLevel(): string | undefined {
   return container.options.level;
 }
 
+/**
+ * Change the global transport of the container which will set default transport for all active loggers.
+ * e.g., to set the global transport call `setGlobalTransports(httpTransport)`.
+ * @param customTransports - The transport to set the global transport to. Both single transport and an array with multiple transpots are supported.
+ */
+export function setGlobalTransports(
+  customTransports: TransportStream | TransportStream[]
+): void {
+  container.options.transports = customTransports;
+  const isArray = (
```

Comment:
[req] I don't think this method is needed. In the if statement you can already check with the `Array.isArray(customTransports)`.
[pp] I would also just use `?:` instead of the if else in that case.

Reply:
thanks. I removed the type predicates and just use the condition with a ternary operator.